src/RA_AchievementOverlay.cpp

```diff
@@ -1123,192 +1078,188 @@ void AchievementOverlay::Render(HDC hRealDC, RECT* rcDest) const
     if (!RAUsers::LocalUser().IsLoggedIn())
         return;	//	Not available!
 
-    if (m_nTransitionState == TS_OFF)
+    if (m_nTransitionState == TransitionState::Off)
         return;
 
-    const int nOuterBorder = 4;
-    RECT rcTarget = (*rcDest);
-    const int nBorder = 8;
-
-    const int nFontSize1 = 32;
-    const int nFontSize2 = 26;
-    const int nFontSize3 = 22;
-    const int nFontSize4 = 16;
-
-    const int nPixelWidth = (*rcDest).right - (*rcDest).left;
-    const BOOL bHiRes = (nPixelWidth >= 320);
-
-    unsigned int nMinUserFrameWidth = 300;
-    unsigned int nMinUserFrameHeight = 64 + 4 + 4;
-    char buffer[1024];
-
-    const int nHeight = rcTarget.bottom - rcTarget.top;
-
+    RECT rcTarget{ *rcDest };
+    const auto lHeight{ rcTarget.bottom - rcTarget.top };
+    {
+        _CONSTANT_LOC nFontSize1{ 32 };
+        _CONSTANT_LOC nFontSize2{ 26 };
+        _CONSTANT_LOC nFontSize3{ 22 };
+        _CONSTANT_LOC nFontSize4{ 16 };
+        _CONSTANT_LOC FONT_TO_USE{ _T("Tahoma") };
```

Comment:
The sheer size of this method needed to be broken up. I didn't want to break anything right now so they are scoped. Those scopes could probably be member/helper functions depending on whether they need access to the class's private members.